src/org/jetbrains/plugins/scala/caches/HydraArtifactsCache.scala

```diff
@@ -0,0 +1,41 @@
+package org.jetbrains.plugins.scala.caches
+
+import java.util.concurrent.ConcurrentHashMap
+
+import org.jetbrains.plugins.scala.project.Platform
+import org.jetbrains.plugins.scala.project.template.Downloader
+
+/**
+  * @author Maris Alexandru
+  */
+object HydraArtifactsCache {
+  private val cache = new ConcurrentHashMap[(String, String), Seq[String]]()
+  private val SPLIT_REGEX = "\\[info\\] \\* Attributed\\(|\\)"
```

Comment:
The Scala convention for declaring constants is to use upper camel case, i.e., SplitRegex instead of SPLIT_REGEX. But maybe you have used `SPLIT_REGEX` because it's the IntelliJ convention?

Reply:
I thought that Scala conventions are the same as Java conventions for declaring constants.